config/dbSetup.sh

```diff
@@ -47,6 +47,9 @@ if [ "$OPEN_WHISK_DB_PROVIDER" == "Cloudant" ]; then
         echo "Cloudant password is not set"
         exit 1
     fi
+
```

Comment:
in `whisk.properties`

```
if [ "$OPEN_WHISK_DB_PROVIDER" == "Cloudant" ]; then
    # Cloudant
    echo "db.host=$CLOUDANT_USERNAME.cloudant.com" >> "$WHISK_HOME/whisk.properties"
    echo "db.port=443" >> "$WHISK_HOME/whisk.properties"
    echo "db.username="$CLOUDANT_USERNAME >> "$WHISK_HOME/whisk.properties"
    echo "db.password="$CLOUDANT_PASSWORD >> "$WHISK_HOME/whisk.properties"
elif [ "$OPEN_WHISK_DB_PROVIDER" == "CouchDB" ]; then
    # CouchDB
    echo "db.host="$COUCHDB_HOST >> "$WHISK_HOME/whisk.properties"
    echo "db.port="$COUCHDB_PORT >> "$WHISK_HOME/whisk.properties"
    echo "db.username="$COUCHDB_USERNAME >> "$WHISK_HOME/whisk.properties"
    echo "db.password="$COUCHDB_PASSWORD >> "$WHISK_HOME/whisk.properties"
```

should these be normalized to use the properties from this script since it sets all the required values?